.travis.yml

```diff
@@ -6,26 +10,38 @@ jdk:
 notifications:
   email: false
 
-sudo: required
-
-before_install:
-  - sudo apt-get -qq update
-  - sudo apt-get install -y pv
-
 env:
   global:
     # Make sure there is enough memory to complete the build
-    - MAVEN_OPTS="-Xms1024m -Xmx2048m -Xss1m -XX:+UseConcMarkSweepGC -XX:+CMSClassUnloadingEnabled"
+    - MAVEN_OPTS="-Xms256m -Xmx512m -Xss16m -XX:+UseConcMarkSweepGC -XX:+CMSClassUnloadingEnabled"
```

Comment:
This is just for the Maven process. The memory for the Scalatest process which runs Flink is specified in the top-level `pom.xml`.